recipes/gromacs/build.sh

```diff
@@ -0,0 +1,149 @@
+#!/bin/bash
+
+# gromacs > 2021 requires a non default OSX version
+if [ "$(uname)" = 'Darwin' ] ; then
+    export MACOSX_DEPLOYMENT_TARGET=10.13 
+fi
+
+
+mkdir build
+cd build
+
+## See INSTALL of gromacs distro
+for ARCH in SSE2 AVX_256 AVX2_256; do
```

Comment:
In Bioconda, CI time was limiting, so we opted not to build an AVX_512 flavor. We could consider relaxing that here. But before we do so, we should also consider whether we plan to add a CUDA build. (I forget why that wasn't done in bioconda, but perhaps the reason was that the CUDA support was not as good as in conda-forge.)

Reply:
I believe it was the lack (at the time) of CUDA support in the bioconda build system which did stop us adding that support.

Reply:
Ok. Nothing *needs* to change here, but AVX_512 is desirable at some future time